localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -36,9 +37,6 @@
 
  innovation_elite_ranks:0 "Elite Ranks"
  innovation_elite_ranks_desc:0 "Our ancestors did not survive through sheer will alone, we will always need the strongest of us defending those who can't defend themselves."
-
- innovation_epidemic:0 "Epidemic"
- innovation_epidemic_desc:0 "The plague spreads like a wildfire, taking countless lives and turning them into the ranks of undead."
```

Comment:
Don't remove or move loc lines in the middle of the file.  I see that German is updated to this, but not French. So update French.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/115126886-7092e600-9fe3-11eb-8d97-252508a83710.png)

</details>